components/scream/src/dynamics/homme/dynamics_driven_grids_manager.cpp

```diff
@@ -50,6 +52,11 @@ DynamicsDrivenGridsManager (const ekat::Comm& comm,
 
   // TODO: add other rebalancing?
 
+  // Get the ref grid name
+  m_ref_grid_name = p.get<std::string>("Reference Grid");
+  EKAT_REQUIRE_MSG (ekat::contains(gn,m_ref_grid_name),
```

Comment:
What does it mean that a valid reference grid name **contains** `m_ref_grid_name`?

Reply:
Well, I don't want one to request XYZ as a reference grid, when XYZ is not even a valid grid name...